tests/utils/syncify.py

```diff
@@ -0,0 +1,19 @@
+import asyncio
+from types import CoroutineType
+from typing import Any, Coroutine, Union
+
+
+def _call(coro: CoroutineType) -> Any:
+    try:
+        loop = asyncio.get_running_loop()
+    except RuntimeError:
+        return asyncio.run(coro)
+    else:
+        return loop.run_until_complete(coro)
+
+
+def run_sync(obj: Union[Coroutine[Any, Any, Any], Any]) -> Any:
```

Comment:
What are the types that actually appear here?

Reply:
It's either the return type of any of our API methods, or a Coroutine with the return type of any of our API methods.

Reply:
Pretty broad. In fact, does that union basically collapse down to `Any`? :)

Reply:
Haha, well I guess so 😭